src/technologies/n.json

```
@@ -130,6 +130,26 @@
     ],
     "website": "https://www.nagish.li"
   },
+  "Naive UI": {
+    "cats": [
+      66
+    ],
+    "description": "Naive UI is a Vue.js UI library written in TypeScript, offering more than 80 treeshakable components.",
+    "icon": "Naive UI.svg",
+    "implies": [
+      "Vue.js",
+      "TypeScript"
+    ],
+    "dom": {
+      "style[cssr-id*='n-']": {
```

If you want a 'starts with' selector (which I think is the case from the code, then you can use this expression.

```suggestion
      "style[cssr-id^='n-']": {
```

---

Hey mate, yes, fixed, thanks!